modal/exception.py

```diff
@@ -42,6 +42,12 @@ class InvalidError(Error):
     """Raised when user does something invalid."""
 
 
+class LocalFunctionError(InvalidError):
```

Comment:
super minor preference for defining this in `_function_utils.py` since it's never used elsewhere, but important